src/twisted/internet/test/test_endpoints.py

```diff
@@ -3259,6 +3259,29 @@ def test_sslNoTrailingNewlinePem(self):
         ctx = server._sslContextFactory.getContext()
         self.assertIsInstance(ctx, ContextType)
 
+    @skipIf(skipSSL, skipSSLReason)
+    def test_sslWithCustomCipher(self):
+        """
+        Test adding custom cipher to the serverFromString function.
```

Comment:
Maybe something like this.

See https://jml.io/test-docstrings/

```suggestion
        A cipher list is supported, using the OpenSSL format.
        The colon (:) from the OpenSSL format is replaced with a comman (').
```